src/package/nuspec/Microsoft.TestPlatform.TestHost.NetCore.props

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Condition=" '$(Platform)' == 'x86' AND '$(OS)' == 'Windows_NT'">
```

Comment:
Why is testhost.x86.exe copied only in case if Platform set to x86? dotnet publish prepare output that can be executed with dotnet vstest, which will get Platfrom parameter at runtime. If both testhost will be in output folder, it will be able to run test with appropriate bitness. Should separate bug for it be created?

Reply:
Yes, thx.

Reply:
Looks like it is last thing to allow resolve #1128, so I've mentioned it there with new comment instead of creating new issue.